<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadMessageAboutDialog</class>
 <widget class="QWidget" name="DownloadMessageAboutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>关于天天下载</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>420</width>
     <height>280</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>4</y>
     <width>350</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>关于天天下载</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>402</x>
     <y>6</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QWidget" name="backgroundMask" native="true">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>29</y>
     <width>420</width>
     <height>255</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{color:#666666;background:white;}
QLabel{background:transparent;}</string>
   </property>
   <widget class="QLabel" name="iconLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>420</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>165</y>
      <width>410</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:15px</string>
    </property>
    <property name="text">
     <string>Mail:Greedysky@163.com greedyskyplus@google.com
Copyright© 2015-2022 Greedysky All Rights Reserved
本软件仅供学习交流，不可用于商业用途，否则后果自负。</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
